packages/functional-tests/tests/syncV3/fxDesktopHandshake.spec.ts

```diff
@@ -0,0 +1,274 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect, newPagesForSync } from '../../lib/fixtures/standard';
+import { FirefoxCommand, createCustomEventDetail } from '../../lib/channels';
+
+import uaStrings from '../../lib/ua-strings';
+
+const password = 'passwordzxcv';
+let syncBrowserPages;
+let browserSignedInEmail;
+let browserSignedInAccount;
+
+let otherEmail;
+let otherAccount;
+
+const ensureUsers = async (page, target, login) => {
+  // Ensure browserSignedInAccount
+  if (!browserSignedInAccount) {
+    browserSignedInEmail = login.createEmail();
+    await target.auth.signUp(browserSignedInEmail, password, {
+      lang: 'en',
+      preVerified: 'true',
+    });
+    (_browserSignedInAccount) => {
+      browserSignedInAccount = _browserSignedInAccount;
+      browserSignedInAccount.email = browserSignedInEmail;
+      browserSignedInAccount.verified = true;
+      login.clearCache();
+    };
+  }
+  // Ensure otherAccount
+  if (!otherAccount) {
+    otherEmail = login.createEmail();
+    await target.auth.signUp(otherEmail, password, {
+      lang: 'en',
+      preVerified: 'true',
+    });
+    (_otherAccount) => {
+      otherAccount = _otherAccount;
+      otherAccount.email = otherEmail;
+      otherAccount.verified = true;
+    };
+  }
+};
+
+test.describe.configure({ mode: 'parallel' });
+
+test.describe('Firefox desktop user info handshake', () => {
+  test.beforeEach(async ({ target, page }) => {
+    test.slow();
+    syncBrowserPages = await newPagesForSync(target);
+    const { login } = syncBrowserPages;
+    await login.clearCache();
+    await ensureUsers(page, target, login);
+  });
+
+  test.afterEach(async () => {
+    await syncBrowserPages.browser?.close();
```

Comment:
`syncBrowserPages.browser?.close()` This expression accesses the close() function of the browser object, which is part of the syncBrowserPages object. The `?.` operator is called the optional chaining operator and is used to prevent errors if the browser object is null or undefined. If the browser object is defined, the `close()` function will be called.